.github/workflows/bump-elastic-stack-snapshot.yml

```diff
@@ -29,15 +28,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Setup Git
-        uses: elastic/apm-pipeline-library/.github/actions/setup-git@current
-
-      - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@453502948b442d7b9a923de7b40cc7ce8628505c
-
-      - name: Run Updatecli
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-          BRANCH: ${{ matrix.branch }}
```

Comment:
The `BRANCH` environment variable isn't defined anymore.

Reply:
good catch, solved in https://github.com/elastic/fleet-server/pull/2345/commits/6ca968091054ef740eaa8f233b5b85e1b838338c